### PR TITLE
Up19 83 change org

### PR DESCRIPTION
**Pull request by larsonme**
This implements the ability for a user to change the organization they are currently impersonating from the header.

### CODE REVIEW

File: packages/app/pages/Organization/orgCard.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ class CardFront extends React.Component {
     let org_image = org.image;
     let card_theme = themable(cardTheme);
     let alt_desc = require("./defaultuserimg.jpg"); // Way of using images in react it has to be done this way for web pack to recognize it. Doing reacts way does not work, only the webpack specific way works.
-    console.log("DEFAULT IMG    "+ alt_desc);
+    // console.log("DEFAULT IMG    "+ alt_desc);
```
Comment (murphyad) [inline packages/app/pages/Organization/orgCard.tsx:19]: Does this need to stay commented, or should it just be removed?

File: packages/app/pages/Organization/organizationCatalog.tsx [hunk 2]
```diff
@@ -67,8 +67,13 @@ class SearchResults extends React.Component {
 
 
   render() {
+    let currentOrganization = -1;
+    if(!isNullOrUndefined(this.props)) {
+      // currentOrganization = this.props.location.state.organizationId;
+    }
     let page_theme = themable(pageTheme);
     let msoeCC = {latitude: 43.044004, longitude: -87.909020};
+    // console.log("Whole thing: " + JSON.stringify(this.props, undefined, 4));
```
Comment (murphyad) [inline packages/app/pages/Organization/organizationCatalog.tsx:76]: Can this be removed?

File: packages/app/pages/Organization/organizationCatalog.tsx [hunk 3]
```diff
@@ -83,7 +88,7 @@ class SearchResults extends React.Component {
         for (let org of organizationList) {
           if (isNullOrUndefined(this.state.nameFilter) || org.name.includes(this.state.nameFilter)) {
             filteredOrgs.push(org);
-            console.log("Org matching filter " + this.state.nameFilter + ": " + org.name);
+            // console.log("Org matching filter " + this.state.nameFilter + ": " + org.name);
```
Comment (murphyad) [inline packages/app/pages/Organization/organizationCatalog.tsx:91]: Remove

**Comment by DonalMoloneyMSOE** [comment]
OrganizationCatalog.ts and header.scss have some commented out code, that could be removed. Also line 71 of organizationCatalog has an if statement with commented code inside of the if. Maybe take out the if statement. Otherwise everything looked fine.

**Review by murphyad — CHANGES_REQUESTED** [review]
A few comments need to be removed, and when the page is changed/refreshed the current org resets to the default value (no org selected)

**Review by DonalMoloneyMSOE — APPROVED** [review]
OrganizationCatalog.ts and header.scss have some commented out code, that could be removed. Also line 71 of organizationCatalog has an if statement with commented code inside of the if. Maybe take out the if statement. Otherwise everything looked fine.

**Pull request closed without merging**